### DISCUSSION TITLE
Unable to get calibration FIL from faucet

### DISCUSSION THREAD

**Discussion by roundbear** [filecoin-project/lotus#6538] [General Q&A] [answered]
Can you send me some calibration FIL？wallet address is：

t3tfhhbre7rahwf26dersq3c4rcniad5mxq4b3byu3ayp6uemu7k27y7kjlo27chps3hgv4n2qsbqpbejfmhna

![image](https://user-images.githubusercontent.com/17848210/122759915-16f1b500-d2cd-11eb-867e-822c2001c8ad.png)

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment, marked as answer]
the site needs a solved google captcha, if you re in china you will need a vpn

**Reply by dd45e640b42e6da7da96faee3996ef7c** [reply]
there are other limits to sending testnet FIL from the faucet: 

- a per wallet limit
- a per IP address limit
